### PR TITLE
draft for new release

### PR DESCRIPTION
**Pull request by biplamal**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -124,7 +124,7 @@ Before proceeding, please ensure [system prerequisites](docs/prerequisites.md) a
 * Optionally, Generate Traffic Using [curl](https://curl.se/)
 
   >We can also pass equivalent **JSON configuration** directly to ixia-c controller, without installing snappi, using **curl**.  
-  >The detailed description of each node (and their attributes) in JSON configuration are well documented [here](https://redocly.github.io/redoc/?url=https://github.com/open-traffic-generator/models/releases/download/v0.7.3/openapi.yaml).
+  >The detailed description of each node (and their attributes) in JSON configuration are well documented [here](https://redocly.github.io/redoc/?url=https://github.com/open-traffic-generator/models/releases/download/v0.7.8openapi.yaml).
```
Comment (rudranil-das) [inline readme.md:127]: please correct typo in the url

File: docs/releases.md [hunk 2]
```diff
@@ -1,6 +1,54 @@
 # Ixia-c Release Notes and Version Compatibility
 
-## Release  v0.0.1-2678 (Latest)
+## Release  v0.0.1-2727 (Latest)
+> 24th March, 2022
+
+#### About
+
+This build implements received(rx) rate statistics of port metrics. It also has auto destination mac learning support using choice as `auto` in ethernet destination mac of packet header.
```
Comment (rudranil-das) [inline docs/releases.md:8]: we can avoid duplication of information - addition of comment to refer to the "New features" section for added functionalities should suffice.

File: docs/releases.md [hunk 3]
```diff
@@ -1,6 +1,54 @@
...
+
+#### Build Details
+
+| Component                     | Version       |
+|-------------------------------|---------------|
+| Open Traffic Generator API    | [0.7.8](https://redocly.github.io/redoc/?url=https://github.com/open-traffic-generator/models/releases/download/v0.7.8/openapi.yaml)         |
+| snappi                        | [0.7.18](https://pypi.org/project/snappi/0.7.18)        |
+| gosnappi                      | [0.7.18](https://pkg.go.dev/github.com/open-traffic-generator/snappi/gosnappi@v0.7.18)        |
+| ixia-c-controller             | [0.0.1-2727](https://hub.docker.com/r/ixiacom/ixia-c-controller/tags)    |
```
Comment (rudranil-das) [inline docs/releases.md:17]: update latest controller build when available

**Comment by rudranil-das** [comment]
rest LGTM

File: readme.md (hunk 1 above)
Comment (biplamal) [reply readme.md]: done

File: docs/releases.md (hunk 3 above)
Comment (biplamal) [reply docs/releases.md:20]: sure will do that

**Pull request merged**